core/controllers/acl_decorators_test.py

```diff
@@ -140,6 +146,14 @@ class PlayCollectionDecoratorTests(test_utils.GenericTestBase):
 
     class MockHandler(base.BaseHandler):
         GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+        URL_PATH_ARGS_SCHEMAS = {
+            'collection_id': {
+                'type': 'unicode'
+            }
+        }
+        HANDLER_ARGS_SCHEMAS = {
+            'GET': {}
+        }
```

Comment:
```suggestion
        HANDLER_ARGS_SCHEMAS = {'GET': {}}
```
Ditto elsewhere so that it is similar throughout the file.

Reply:
Mentioned a comment below, for not removng schema ptal.